src/site/content/en/index.md

```diff
@@ -14,11 +14,11 @@ intro:
 
 featuredCard:
   eyebrow: 'Featured'
-  title: 'Watch the progress we made together on the web.'
+  title: 'Let’s go. It’s Google I/O. Register Now.'
   background: ''
-  url: 'https://www.youtube.com/watch?v=W9O4CuSUKb8'
-  video: 'https://web-dev.imgix.net/video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/lIta9ZJEALbfAzJzmiNl.mp4'
-  alt: 'Chrome icon rolls past a title: 2022 Year in Review.'
+  url: 'https://io.google/2023/?utm_source=web&utm_medium=embedded_marketing&utm_campaign=hpp_reg_card&utm_content='
+  image: 'image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/t6UJBGxTD5XYwq8VgsK5.png'
```

Comment:
Sorry, it's confusing. `image` is only considered if `video` is set and then defines a fallback image. For a pure image featured card one is expected to use `background`. Then it looks like this:

<img width="502" alt="image" src="https://user-images.githubusercontent.com/12857772/225386001-053f81c2-a5b2-4412-8019-4818e353e3c0.png">

Not ideal for a11y reasons.

Reply:
Thanks Matthias. We had to do some reconfig to get a video to appear there in the first place so I think I was a little confused.